src/_includes/common/filter-events.md

```diff
@@ -0,0 +1,129 @@
+<!--
+Guideline: This page is common to all SDKs; it is stored in the common folder, nested under _includes/common. To use, 
+
+1. Add a folder with the name of the platform you are documenting to the _documentation/sdks structure (for example, _documentation/sdks/javascript) 
+2. Create a new filter.md file in _documentation/sdks/<platform-name>/config folder
+3. Create the defined `include` statements and add them to the filter.md file
+
+If you have questions, please ask Fiona or Daniel. 
+
+**The objective for this page is that a developer can view easily the configuration options for the SDK; each page _must_ have a description that includes a summary of what the page provides to the developer. Simply linking the page is insufficient.**
+-->
```

Comment:
These comments will show up in the source code, maybe we'll find a better place for them or use `{% comment %}` `{% endcomment %}` instead of `<!--` `-->`

Reply:
Done. Thanks.